scripts/release_scripts/deploy.py

```diff
@@ -612,7 +613,7 @@ def execute_deployment():
 
         python_utils.PRINT('Done!')
     finally:
-        common.run_cmd([
+        subprocess.check_output([
```

Comment:
Perhaps explain the rationale behind this change?

Reply:
Just to be consistent - everywhere else in the main method subprocess.check_output is being used.